src/exts/replay_handler.js

```diff
@@ -1,26 +1,11 @@
-const path = require('path');
-
-const { DemoParser } = require('sdfz-demo-parser');
-
 const { bridge } = require('../spring_api');
 const springPlatform = require('../spring_platform');
 const { log } = require('../spring_log');
+const { parseReplay } = require('../replays_utils');
```

Comment:
I'd prefer having just one plural in file names, so either name this "replay_util" or "replay_utils"